docs/learn/digital-assets/read-asset-data.md

```diff
@@ -267,744 +302,79 @@ console.log(assetMetadata);
-
-```js
-// ...
-
-function decodeTokenId(encodedTokenId: string, tokenIdFormat: number) {
-  switch (tokenIdFormat) {
-    // Number
-    case 0:
-    case 100:
-      return BigInt(encodedTokenId).toString();
-    // String
-    case 1:
-    case 101:
-      return web3.utils.hexToUtf8(encodedTokenId).replace(/\0+$/, '');
-    // Address
-    case 2:
-    case 102:
-      return '0x' + encodedTokenId.slice(encodedTokenId.length - 40);
-    // Byte Value
-    case 3:
-    case 103:
-      // Extracts the non-zero portion (right padded)
-      return encodedTokenId.replace(/0+$/, '');
-    // Hash Digest
-    case 4:
-    case 104:
-      // Hash digests are not modified during encoding, so return as is
-      return encodedTokenId;
-  }
+  key: '0x114bd03b3a46d48759680d81ebb2b414fda7d030a7105a851867accf1c2352e7',
+  name: 'LSP4Creators[]',
+  value: []
 }
-
-// Decode the token ID based on the token ID format
-const decodedTokenId = decodeTokenId(byte32TokenId, tokenIdFormat);
-```
-
-  </TabItem>
-
-</Tabs>
-
-</details>
-
-```js
-// ...
-
-// Build link to JSON metadata
-const baseURLlink = decodedBaseURI[0].value.url;
-const metadataJsonLink = `${baseURLlink}${decodedTokenId}`;
-
-// Fetch the URL
-const response = await fetch(metadataJsonLink);
-const jsonMetadata = await response.json();
-console.log('Metadata Contents: ', jsonMetadata);
-```
-
-  </TabItem>
-
-</Tabs>
-
-<details>
-<summary>Show result</summary>
-
-```js
-{
-    "LSP4Metadata": {
-      "name": "My Token Name",
-      "description": "Sample Description",
-      "links": [{ "title": "", "url": "" }],
-      "icon": [
-        {
-          "width": 1600,
-          "height": 1600,
-          "url": "ipfs://QmRHz3nbd2wQ4uNCJQS9JDnZRdD6aqueWRZ2h89dMkCLXf"
-        }
-      ],
-      "images": [
-        [
-          {
-            "width": 1000,
-            "height": 1000,
-            "url": "ipfs://QmRHz3nbd2wQ4uNCJQS9JDnZRdD6aqueWRZ2h89dMkCLXf",
-            "verification": {
-              "method": "keccak256(bytes)",
-              "data": "0x7e89fc626703412916d27580af2bae16db479036f12d4d48f4efd24d70224dc2"
-            }
-
-          }
-        ]
-      ],
-      "assets": [],
-      "attributes": [
-        {
-          "key": "Standard type",
-          "value": "LSP",
-          "type": "string"
-        },
-        {
-          "key": "Standard number",
-          "value": 4,
-          "type": "number"
-        }
-      ]
-    }
-  }
+*/
 ```
 
-</details>
-
-## Fetch Creators and Token Properties
+### Get Creator basic infos
 
-Instead of using the [`LSP4Metadata`](../../standards/tokens/LSP4-Digital-Asset-Metadata.md) key to retrieve the [global token information](#global-token-information), you can also retrieve other data keys as described in the examples below. The full setup using [erc725.js](../../tools/erc725js/getting-started.md) can be found within the [setup section](#setup).
+:::success Advice
```

Comment:
I think we need agreement on using these. I like "advice" but maybe we should standardise them and just using :::info